### PR TITLE
[WIP] Begin implementation of PID Arcade Drive

### LINKED ISSUE #11: Create arcadeDrive() method in the Drive class
Use encoder read outs 
image
and the encoder section on the follow page
https://frc-pdr.readthedocs.io/en/latest/control/driving_straight.html

to create a p loop to drive the difference between the left and right side encoders to zero. This is to ensure straight driving

### PR DESCRIPTION
**Pull request by Rene-Baquero**
resolves #11

### CODE REVIEW

File: src/main/java/org/usfirst/frc1745/deepspace2019/Robot.java [hunk 1]
```diff
@@ -122,14 +122,14 @@ public void teleopInit() {
...
-        drive.setLeftSpeed(-controls.getLeftY(DEADZONE) * .75);
-        drive.setRightSpeed(controls.getRightY(DEADZONE) * .75);
+        drive.arcadeDrive(controls.getLeftY(DEADZONE), controls.getRightX(DEADZONE));
 
         //Sets network table for limelight
         NetworkTable limelightNetworkTable = NetworkOperations.getNetworkTable("limelight");
         double[] calculatedDeltas = limelight.calcSpeed(limelightNetworkTable);
 
         getSetLimelightValues(calculatedDeltas, limelightNetworkTable);
+        drive.setkI(NetworkOperations.getPreferencesDouble("KiConstant"));
```
Comment (kushagharahi) [inline src/main/java/org/usfirst/frc1745/deepspace2019/Robot.java:142]: 💯

**Comment by kushagharahi** [comment]
@Rene-Baquero I added the relevant story link in your PR description so they get linked in the github kanban. Let's start doing that from now on

**Comment by kushagharahi** [comment]
image
This is what that updated description does now. 
https://github.blog/2013-05-14-closing-issues-via-pull-requests/
image

**Review by nbgray — COMMENTED** [review]
Delete PRs this old

**Pull request closed without merging**